src/main/java/com/crud/tasks/controller/TaskController.java

```diff
@@ -0,0 +1,38 @@
+package com.crud.tasks.controller;
+
+import com.crud.tasks.domain.TaskDto;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/task")
+public class TaskController {
```

Comment:
change implementation as I suggested in private message (https://hackernoon.com/restful-api-designing-guidelines-the-best-practices-60e1d954e7c9)